tests/python/MypyBearTest.py

```diff
@@ -0,0 +1,124 @@
+from queue import Queue
+from textwrap import dedent
+
+from bears.python.MypyBear import MypyBear
+from tests.BearTestHelper import generate_skip_decorator
+from tests.LocalBearTestHelper import LocalBearTestHelper
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.misc.ContextManagers import prepare_file
+
+
+@generate_skip_decorator(MypyBear)
+class MypyBearTest(LocalBearTestHelper):
```

Comment:
you're fiddling a lot with the metadata, can you add a test ensuring all params end up in the joined metadata?